tasks/gitlab_helpers.py

```diff
@@ -91,7 +105,32 @@ def get_link_to_job_on_main(job_name: str):
     }
     query_string = to_query_string(query_params)
     quoted_query_string = quote(string=query_string, safe="")
-    return f"{CI_VISIBILITY_URL}?query={quoted_query_string}"
+    return f"{PIPELINE_VISIBILITY_URL}?query={quoted_query_string}"
+
+
+def get_link_to_test_runs(job_id: str):
+    query_params = {
+        "test_level": "test",
+        "@ci.job.url": f"\"https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/{job_id}\"",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏You can use `BASE_URL` from `tasks/libs/ciproviders/gitlab_api.py` to avoid hard coding the url